src/StateSmith/Output/Sim/HtmlRenderer.cs

```diff
@@ -108,6 +108,23 @@ public static void Render(StringBuilder stringBuilder, string smName, string moc
         display: table-cell;
       }
 
+      table.console td {
+          color: rgba(0, 0, 0, 0.7);
+      }
+
+      table.console td .dispatched {
+          font-weight: bold;
+          color: rgba(0, 0, 0, 1);
+      }
+
+      table.console tr:has(+tr td .dispatched) {
+          border-bottom: 0px;
+      }
+
+      table.console tr:has(+tr td .dispatched) td {
+          padding-bottom: 50px;
```

Comment:
I really like this overall direction :) Looks much more professional. The I think that 50px might be too much space in between though. Here's an image for comparison:

![image](https://github.com/StateSmith/StateSmith/assets/274012/34dbbb39-5923-4a77-8470-1e9c04f08b19)

Thoughts?